setup.py

```diff
@@ -48,6 +48,7 @@
         "amazon-braket-schemas ~= 1.10",
         "amazon-braket-default-simulator ~= 1.9",
         "boto3-stubs",
+        "numpy == 1.23.3",
```

Comment:
If this is only a problem with Windows and Python 3.10 then I'd probably prefer to temporarily disable tests on that combination. Pinning the numpy version is rather intrusive for people who have other packages installed in the same environment. Either way we should keep the issue open until we have a proper fix.

Reply:
Yes I agree on this, I first wanted to check if this solves the problem on the CI, too. Will update this PR.